src/logic/safe/transactions/gasNew.js

```diff
@@ -56,18 +57,18 @@ export const estimateTxGasCosts = async (
       txData = await safeInstance.methods
         .execTransaction(to, tx ? tx.value : 0, data, CALL, 0, 0, 0, ZERO_ADDRESS, ZERO_ADDRESS, signatures)
         .encodeABI()
+      gas = await calculateGasOf(txData, from)
```

Comment:
https://web3js.readthedocs.io/en/v1.2.0/web3-eth.html#id62

to is a required parameter, it can be dropped only for contract creation transactions, so this is not the fix

I'd suggest looking into the arguments we pass to estimation function inside the react component, maybe we forget to pass safeTxGas or something